server/src/main/java/com/objectcomputing/CheckInsRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.objectcomputing;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.objectcomputing.checkins.CheckIns;
+
+public interface CheckInsRepository {
+
+    List<CheckIns> findByName(String teamMember);
+    List<CheckIns> findByTargetQtr(String targetQtr);
+    List<CheckIns> findByPdlId(UUID pdlId);
+    List<CheckIns> findByTargetYear(String targetYear);
```

Comment:
I'm think the target year and quarter have to be one call.  Perhaps this is just targetQuarter and that value should include year (i.e. "2020Q1")?